src/DurableTask.Netherite/Scaling/ScalingMonitor.cs

```diff
@@ -249,6 +247,22 @@ public async Task<string> TaskHubIsIdleAsync(Dictionary<uint, PartitionLoadInfo>
 
             // we have concluded that there are no pending work items, timers, or unprocessed input queue entries
             return null;
-        }  
+        }
+
+        public static async Task<List<long>> GetQueuePositionsAsync(ConnectionInfo connectionInfo, string partitionHub)
```

Comment:
The method added to query the partitions Info in eventhubs for scale monitor.  This method is the substitution of method[ `GetQueuePositionAsync`](https://github.com/microsoft/durabletask-netherite/blob/ba15eed262d72741bb52b45fb8a71137192c24b6/src/DurableTask.Netherite/TransportLayer/EventHubs/EventHubsConnections.cs#L228) in `EventHubsConnections`.